src/test/java/uk/gov/hmcts/cmc/claimstore/events/paindinfull/PaidInFullCitizenNotificationHandlerTest.java

```diff
@@ -42,17 +45,44 @@ public void setUp() {
     }
 
     @Test
-    public void sendNotificationsSendsNotificationsToDefendant() throws NotificationClientException {
-
-        PaidInFullEvent event = new PaidInFullEvent(SampleClaim.getDefault());
+    public void sendNotificationsSendsNotificationsToDefendantEmailNotLinked() throws NotificationClientException {
+        Response fullDefenceResponse = SampleResponse.FullDefence.builder().build();
```

Comment:
We don't need `Response` as defendant is not linked yet. We just need a claim submitted by claimant here.